health.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package mainflux
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+const (
+	contentType     = "Content-Type"
+	contentTypeJSON = "application/health+json"
+	svcStatus       = "pass"
+	description     = " service"
+)
+
+var (
+	// Version represents the last service git tag in git history.
+	// It's meant to be set using go build ldflags:
+	// -ldflags "-X 'github.com/mainflux/mainflux.Version=0.0.0'"
+	Version = "0.0.0"
+	// Commit represents the service git commit hash.
+	// It's meant to be set using go build ldflags:
+	// -ldflags "-X 'github.com/mainflux/mainflux.Commit=ffffffff'"
+	Commit = "ffffffff"
+	// BuildTime represetns the service build time.
+	// It's meant to be set using go build ldflags:
+	// -ldflags "-X 'github.com/mainflux/mainflux.BuildTime=1970-01-01_00:00:00'"
+	BuildTime = "1970-01-01_00:00:00"
+)
+
+// HealthInfo contains version endpoint response.
+type HealthInfo struct {
+	// Status contains service status.
+	Status string `json:"status"`
+
+	// Version contains current service version.
+	Version string `json:"version"`
+
+	// Commit represents the service build time.
```

Comment:
This is not correct. It is a Git commit hash.

Reply:
right

Reply:
Please fix